### ISSUE TITLE
Version 1.0.62: Starting the Server doesn't work with some settings

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#7]
```
What steps will reproduce the problem?
Starting the Server tool with the following settings does not work in
version 1.0.62:

java org.h2.tools.Server -tcpAllowOthers true -webAllowOthers true

What is the expected output? What do you see instead?

I used to work, now the possible options are listed.

```

Original issue reported on code.google.com by `thomas.t...@gmail.com` on 26 Nov 2007 at 5:35

**Comment by GoogleCodeExporter** [comment]
```
As a workaround, remark lines 157-159 in /src/main/org/h2/tools/Server.java:

// } else {
//  showUsage();
//  return EXIT_ERROR;

...and then recompile H2:

ant jar

```

Original comment by `thomas.t...@gmail.com` on 26 Nov 2007 at 5:39

**Comment by GoogleCodeExporter** [comment]
```
Hi!
Have done so, and got errors (see later). Maybe you just make a patched version 
instead? :-) P.S. Thanks for a such nice product! :-)
>ant jar
Buildfile: build.xml

init:
     [echo] build-jdk:1.6 ant-build.properties/jdk:1.4

clean:

compileTest:
     [echo] Java version is 1.6 but source code is switched to 1.4.
     [echo] Run ant codeswitchJdk... first.

compileResources:
    [javac] Compiling 1 source file to C:\bin\h2\bin
    [javac] C:\bin\h2\src\main\org\h2\jdbc\JdbcConnection.java:56: 
org.h2.jdbc.JdbcConnection is not abstract and does not override abstract 
method 
createStruct
(java.lang.String,java.lang.Object[]) in java.sql.Connection
    [javac] public class JdbcConnection extends TraceObject implements Connection {
    [javac]        ^
    [javac] C:\bin\h2\src\main\org\h2\jdbc\JdbcStatement.java:26: 
org.h2.jdbc.JdbcStatement is not abstract and does not override abstract method 
isWrapperFor(java.lang.Class) in java.sql.Wrapper
    [javac] public class JdbcStatement extends TraceObject implements Statement
{
    [javac]        ^
    [javac] C:\bin\h2\src\main\org\h2\jdbc\JdbcResultSet.java:68: 
org.h2.jdbc.JdbcResultSet is not abstract and does not override abstract method 
updateNClob(java.lang.String,java.io.Reader) in java.sql.ResultSet
    [javac] public class JdbcResultSet extends TraceObject implements ResultSet
{
    [javac]        ^
    [javac] C:\bin\h2\src\main\org\h2\tools\SimpleResultSet.java:52: 
org.h2.tools.SimpleResultSet is not abstract and does not override abstract 
method 
updateSQLXML(java.lang.String,java.sql.SQLXML) in java.sql.ResultSet
    [javac] public class SimpleResultSet implements ResultSet, ResultSetMetaData
 {
    [javac]        ^
    [javac] C:\bin\h2\src\main\org\h2\jdbc\JdbcPreparedStatement.java:64: 
org.h2.jdbc.JdbcPreparedStatement is not abstract and does not override 
abstract 
method setSQLXML(int,java.sql.SQLXML) in java.sql.PreparedStatement
    [javac] public class JdbcPreparedStatement extends JdbcStatement implements
PreparedStatement {
    [javac]        ^
    [javac] C:\bin\h2\src\main\org\h2\jdbc\JdbcDatabaseMetaData.java:27: 
org.h2.jdbc.JdbcDatabaseMetaData is not abstract and does not override abstract 
method
getRowIdLifetime() in java.sql.DatabaseMetaData
    [javac] public class JdbcDatabaseMetaData extends TraceObject implements 
DatabaseMetaData {
    [javac]        ^
    [javac] C:\bin\h2\src\main\org\h2\jdbc\JdbcCallableStatement.java:38: 
org.h2.jdbc.JdbcCallableStatement is not abstract and does not override 
abstract 
method getSQLXML(java.lang.String) in java.sql.CallableStatement
    [javac] public class JdbcCallableStatement extends JdbcPreparedStatement imp
lements CallableStatement {
    [javac]        ^
    [javac] C:\bin\h2\src\main\org\h2\jdbc\JdbcResultSetMetaData.java:20: 
org.h2.jdbc.JdbcResultSetMetaData is not abstract and does not override 
abstract 
method isWrapperFor(java.lang.Class) in java.sql.Wrapper
    [javac] public class JdbcResultSetMetaData extends TraceObject implements 
ResultSetMetaData {
    [javac]        ^
    [javac] C:\bin\h2\src\main\org\h2\jdbc\JdbcParameterMetaData.java:20: 
org.h2.jdbc.JdbcParameterMetaData is not abstract and does not override 
abstract 
method isWrapperFor(java.lang.Class) in java.sql.Wrapper
    [javac] public class JdbcParameterMetaData extends TraceObject
    [javac]        ^
    [javac] 9 errors

BUILD FAILED
>\build.xml:90: Compile failed; see the compiler error output for details.
```

Original comment by `Ismailov...@gmail.com` on 26 Nov 2007 at 2:13

**Comment by GoogleCodeExporter** [comment]
```
Hi,

The problem is:
>      [echo] Java version is 1.6 but source code is switched to 1.4.
>      [echo] Run ant codeswitchJdk... first.
You need to switch the source code to JDK 1.6 first:

ant codeswitchJdk16

Then you can compile.
Regards,
Thomas
```

Original comment by `thomas.t...@gmail.com` on 26 Nov 2007 at 9:35

**Comment by GoogleCodeExporter** [comment]
```
The problem is resolved in the Subversion repository.
```

Original comment by `thomas.t...@gmail.com` on 26 Nov 2007 at 9:37

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.0.63
```

Original comment by `thomas.t...@gmail.com` on 4 Dec 2007 at 8:08
- Changed state: **Fixed**
